### PR TITLE
[4.0] First round of fixes for the basic template files

### PR DESCRIPTION
**Pull request by zero-24**
### Summary of Changes

First round of fixes for the basic template files

### Testing Instructions

Install still works

### Expected result

Install works

### Actual result

some cs and improvments

### Documentation Changes Required

none

### CODE REVIEW

**Comment by zero-24** [comment]
drone fails are not about this PR.. I'm going to do a new one for this.

File: installation/language/en-GB/en-GB.ini [hunk 1]
```diff
@@ -11,6 +11,10 @@ MIN_PHP_ERROR_TEXT="Your host needs to use PHP version {{phpversion}} or newer t
 MIN_PHP_ERROR_URL_TEXT="Help me resolve this"
 ;End of minimum PHP error page
 
+;Universal strings
+INSTL_JOOMLA_LOGO="The Joomla! Logo"
```
Comment (dgrammatiko) [inline installation/language/en-GB/en-GB.ini]: Please remove the `!` as it messes up screen readers

File: installation/template/error.php [hunk 2]
```diff
@@ -9,86 +9,68 @@
...
+							<span class="fa fa-exclamation" aria-hidden="true"></span>
+							<?php echo JText::_('INSTL_ERROR'); ?>
 						</div>
 						<div class="j-install-step-form">
 							<div class="alert preinstall-alert">
 								<div class="alert-icon">
 									<span class="alert-icon fa fa-exclamation-triangle"></span>
 								</div>
 								<div class="alert-text">
-									<h2><?php echo JText::_('JERROR_LAYOUT_ERROR_HAS_OCCURRED_WHILE_PROCESSING_YOUR_REQUEST'); ?></h2>
```
Comment (dgrammatiko) [inline installation/template/error.php:60]: Erm, why do you want to make this 3 lines?

Comment (zero-24) [reply installation/template/error.php:60]: like we do on all other places ;) `<h2>` opens a tag like `<div>` etc ;)

Comment (dgrammatiko) [reply installation/template/error.php:60]: sorry, but this is stupid

**Pull request closed without merging**